lib/stm32/f4/rcc.c

```diff
@@ -307,6 +357,22 @@ const struct rcc_clock_scale rcc_hse_25mhz_3v3[RCC_CLOCK_3V3_END] = {
 		.apb1_frequency = 42000000,
 		.apb2_frequency = 84000000,
 	},
+    { /* 96MHz */
+		.pllm = 25,
+		.plln = 192,
```

Comment:
Sorry, I should have made this two separate commits. So, I did two
distinct things:

One - add an HSI PLL config convenience function and alias the 16MHz HSE
presets to rcc_hsi_3v3

Two - add a 96MHz preset to ALL the HSE preset sets, including the 25MHz
one you're looking at here.

On 09/05/2016 11:51 AM, Karl Palsson wrote:
> @@ -307,6 +357,22 @@ const struct rcc_clock_scale rcc_hse_25mhz_3v3[RCC_CLOCK_3V3_END] = {
>>  		.apb1_frequency = 42000000,
>>  		.apb2_frequency = 84000000,
>>  	},
>> +    { /* 96MHz */
>> +		.pllm = 25,
>> +		.plln = 192,